### PR TITLE
mailmap updates

### PR DESCRIPTION
**Pull request by swamireddy**

### CODE REVIEW

**Comment by swamireddy** [comment]
@dachary : Updated this PR with all changes to map file. Please check.

**Comment by dachary** [comment]
@swamireddy could you please make it so there is one commit per person instead of clustering all the changes into just two commits ? It is easier to review this way. Thanks !

File: .mailmap [hunk 1]
```diff
@@ -245,13 +252,17 @@ Xan Peng <xanpeng@gmail.com> xan <xanpeng@gmail.com>
 Xavier Roche <roche+git@exalead.com> <roche@httrack.com>
 Xiaowei Chen <chen.xiaowei@h3c.com> <cxwshawn@gmail.com>
 Xie Rui <875016668@qq.com> Jerry7X <875016668@qq.com>
+Xie Rui <875016668@qq.com> Xie Rui <jerry.xr86@gmail.com>
+xiexingguo <258156334@qq.com> <xie.xiexingguo@zte.com.cn>
```
Comment (dachary) [inline .mailmap]: Xie Xiexingguo ?

File: .organizationmap [hunk 2]
```diff
@@ -115,6 +118,7 @@ H3C <wentao@h3c.com> Qiankun Zheng <zheng.qiankun@h3c.com>
 H3C <wentao@h3c.com> Ruifeng Yang <yangruifeng.09209@h3c.com>
 H3C <wentao@h3c.com> Sangdi Xu <xu.sangdi@h3c.com>
 H3C <wentao@h3c.com> Shan Li <li.shan@h3c.com>
+H3C <wentao@h3c.com> shawn <chen.xiaowei@h3c.com>
```
Comment (dachary) [inline .organizationmap]: Chen Xiaowei with the associated .mailmap change ?

File: .organizationmap [hunk 3]
```diff
@@ -321,9 +335,11 @@ Unaffiliated <no@organization.net> Andy Allan <andy@gravitystorm.co.uk>
 Unaffiliated <no@organization.net> Anis Ayari <ayari_anis@live.fr>
 Unaffiliated <no@organization.net> Armando Segnini <armaseg@gmail.com>
 Unaffiliated <no@organization.net> Arthur Gorjux <arthurgorjux@gmail.com>
+Unaffiliated <no@organization.net> blinke <Burkhard.Linke@computational.bio.uni-giessen.de>
```
Comment (dachary) [inline .organizationmap]: Burkhard.Linke ? with the associated .mailmap change ?

**Comment by dachary** [comment]
In **Commits, by organizations** I see
<pre>
    25	      1 minchen &lt;minchen@ubuntukylin.com>
    30	      1 Brian Andrus &lt;bandrus+github@gmail.com>
</pre>
In **Reviews, by organizations* I see
<pre>
    12	      1 Ken Dreyer, &lt;kdreyer@redhat.com>
    13	      1 Joao Eduardo Luis &lt;joao@suse.com>
    14	      1 Jason Dillaman &lt;jdillaman@redhat.com>
    15	      1 James Page &lt;jamespage@ubuntu.com>
</pre>

**Comment by swamireddy** [comment]
@dachary - All changes done. Please check now.

**Comment by swamireddy** [comment]
@dachary  - any issues with patch? Do I need to fix anything here?

**Pull request merged**